charts/kubernikus-system/charts/blackbox-exporter/values.yaml

```diff
@@ -0,0 +1,13 @@
+# Default values for blackbox-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
```

Comment:
Would it make sense to have 2? Or is that a waste of resources...

Reply:
In the end we would have 2. 1 in the kluster, as part of kubernikus-system, and another one calling from the outside, from the admin cluster.